packages/components/src/components/FilterMenu.vue

```diff
@@ -225,6 +307,19 @@ export default {
         ['ruby', 'javascript'].includes(appMap.metadata.language.name)
       );
     },
+
+    defaultButtonClass() {
+      const suffix = this.selectedFilter && this.selectedFilter.default ? '-disabled' : '';
+      return 'filters__button' + suffix;
+    },
+
+    deleteButtonClass() {
+      const suffix =
+        this.selectedFilter && this.selectedFilter.filterName === DEFAULT_FILTER_NAME
```

Comment:
Is this unique?

Reply:
Yes, if you try to save a filter with the same name as a previously saved filter, then the old one gets overwritten. I added logic to prevent that from happening with the `AppMap default` filter.